### PR TITLE
Status endpoints

### PR DESCRIPTION
**Pull request by kompotkot**
## Changes

Status endpoint for:
- db
- crawlers

## How to test these changes?

Tested manually

### CODE REVIEW

File: crawlers/server/main.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	response := PingResponse{Status: "ok"}
+	json.NewEncoder(w).Encode(response)
+}
+
+func pingGeth(w http.ResponseWriter, req *http.Request) {
+	log.Printf("%s, %s, %q", req.RemoteAddr, req.Method, req.URL.String())
+
+	postBody, err := json.Marshal(map[string]interface{}{
+		"jsonrpc": "2.0",
+		"method":  "eth_syncing",
```
Comment (zomglings) [inline crawlers/server/main.go:45]: `eth_syncing` returns `false` if the node is synced. If the node is synchronizing to the blockchain, it returns information about how far it has made progress.

To get current block number, use `eth_blockNumber`.

Most importantly, response on `/status` endpoint should have this format:
```json
{
    "synchronizing": false,
    "synchronizationState": {},
    "blockNumber": 13282394
}
```

Comment (kompotkot) [reply crawlers/server/main.go]: I will add this in next update

File: db/deploy/deploy.bash [hunk 2]
```diff
@@ -5,15 +5,32 @@
...
 APP_DB_SERVER_DIR="${APP_DIR}/db/server"
+SECRETS_DIR="${SECRETS_DIR:-/home/ubuntu/moonstream-secrets}"
 SCRIPT_DIR="$(realpath $(dirname $0))"
 SERVICE_FILE="${SCRIPT_DIR}/moonstreamdb.service"
 
 set -eu
 
+echo
+echo
+echo "Retrieving deployment parameters for GCL Secret Manager"
```
Comment (zomglings) [inline db/deploy/deploy.bash:16]: GCP

**Review by Andrei-Dolgolev — APPROVED** [review]
Looks correct

**Pull request merged**